.env

```diff
@@ -1,3 +1,3 @@
-VUE_APP_EPOCH_URL=https://sdk-mainnet.aepps.com/
+VUE_APP_EPOCH_URL=https://roma-net.mdw.aepps.com/
```

Comment:
There should not be commits from other branches into this PR. You can make a new PR if you want to merge changes from develop into develop-new-ui. It is difficult to review for all these changes.